Code/Source/Camera/CameraCaptureScheduler.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/std/string/string.h>
+#include <AzCore/std/containers/vector.h>
+#include <Atom/Feature/Utils/FrameCaptureBus.h>
+
+#include <Clock/SimulationClock.h>
+
+#include <Atom/RPI.Public/RenderPipeline.h>
+
+#include <list>
+
+namespace ROS2 {
+
+//! Singleton class to initialize and synchronize the frame captures for multiple cameras
+//! It supports following features:
+//! - only one frame is rendered and captured at a time with waiting for capture finished notification
```

Comment:
This sentence is hard to understand at first reading.

Reply:
split?
```suggestion
//! - only one frame is rendered and captured at a time 
//! - scheduler waits for a notification that capture is finished
```